src/content/api/cli.md

```diff
@@ -136,55 +136,81 @@ webpack --env.platform=web  # sets env.platform == "web"
-`--output-public-path`    | The public path for the assets              | string     | /
-`--output-source-map-filename` | The output filename for the SourceMap  | string     | [name].map or [outputFilename].map
-`--build-delimiter` | Display custom text after build output | string | Default string is null. You could provide a string such as `=== Build done ===`
+| Parameter                      | Explanation                                                     | Input type | Default                                                                         |
+| ------------------------------ | --------------------------------------------------------------- | ---------- | ------------------------------------------------------------------------------- |
+| `--output-chunk-filename`      | The output filename for additional chunks                       | string     | filename with [id] instead of [name] or [id] prefixed                           |
+| `--output-filename`            | The output filename of the bundle                               | string     | [name].js                                                                       |
+| `--output-jsonp-function`      | The name of the JSONP function used for chunk loading           | string     | webpackJsonp                                                                    |
+| `--output-library`             | Expose the exports of the entry point as library                | string     |
+| `--output-library-target`      | The type for exposing the exports of the entry point as library | string     | var                                                                             |
+| `--output-path`                | The output path for compilation assets                          | string     | Current directory                                                               |
+| `--output-pathinfo`            | Include a comment with the request for every dependency         | boolean    | false                                                                           |
+| `--output-public-path`         | The public path for the assets                                  | string     | /                                                                               |
+| `--output-source-map-filename` | The output filename for the SourceMap                           | string     | [name].map or [outputFilename].map                                              |
+| `--build-delimiter`            | Display custom text after build output                          | string     | Default string is null. You could provide a string such as `=== Build done ===` |
 
+### Cascade Options
+
+CLI allows to provide variables by adding `--` before the variable name and value. You can access such variables in the configuration file.
+
+For npm:
+
+```bash
+"build": "webpack --mode production --config webpack.prod.config.js"
```

Comment:
these are not bash commands, looks like scripts from package.json declaration, maybe you want to correct it to either be bash command or change it to package.json example? Although we dont really do package.json examples outside of guides so i'd vote for 1st